### PR TITLE
Modify UserGuide language

### PR DESCRIPTION
**Pull request by Haris-Irfan**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -288,7 +286,7 @@ You can add the prefixes in any order you want!
 
 ### Listing all events: `list event`
 
-Shows a list of all events in TalentHub.
+This command allows us to view a list of all events on TalentHub!
```
Comment (jhqiu21) [inline docs/UserGuide.md:289]: Maybe "us" should be you for word consistency?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -387,7 +384,7 @@ Examples:
 
 ### Deleting an event : `delete event`
 
-Deletes the specified event from the address book.
+This command allow you to delete the specified event from the list of events on TalentHub!
```
Comment (jhqiu21) [inline docs/UserGuide.md:387]: I think there "allow" should be "allows".

File: docs/UserGuide.md (hunk 1 above)
Comment (Haris-Irfan) [reply docs/UserGuide.md]: Got it!

File: docs/UserGuide.md (hunk 2 above)
Comment (Haris-Irfan) [reply docs/UserGuide.md]: Done!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -42,7 +40,7 @@ If you can type fast, and you need to manage celebrities and their schedules, **
 
    - `delete person 3` : Deletes the 3rd person shown in the current list after receiving confirmation from you.
 
-   - `clear all` : Deletes all events and persons after receiving confirmation from you.
+   - `clear all` : Deletes all events and persons after receiving confirmation from you. This will permanently delete all data currently saved on TalentHub!
```
Comment (fusin02) [inline docs/UserGuide.md:43]: Should we add this? Bc below alr mentions this

File: docs/UserGuide.md [hunk 4]
```diff
@@ -473,15 +470,15 @@ Example:
...
 
 ### Editing the data file
 
-TalentHub data are saved automatically as a JSON file `[JAR file location]/data/talenthub.json`. Advanced users are welcome to update data directly by editing that data file.
+Your data is saved automatically as a JSON file `[JAR file location]/data/talenthub.json`! If you are experienced with handling such files, you are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 If your changes to the data file makes its format invalid, TalentHub will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
-Furthermore, certain edits can cause TalentHub to behave in unexpected ways (e.g., if a value entered is outside of the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
+Furthermore, certain edits can cause TalentHub to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly!
```
Comment (fusin02) [inline docs/UserGuide.md:481]: Maybe we can be clear and say it'll erase all the stuff and start with empty

**Review by seantham21 — COMMENTED** [review]
lgtm

**Pull request merged**